### DISCUSSION TITLE
FastAPI behaving little strange when I hit the API multiple times. Could you help me to resolve it?

### DISCUSSION THREAD

**Discussion by Samba-Odoo-Apps** [fastapi/fastapi#9545] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class GetDataModelOut(BaseModel):
    count: int = 1
    next: str = None
    previous: str = None
    result: List[dict] = [{}]
    message: str = "Records Found Successfully"
    id: str = None
    statusCode: int = 200

class Emp:
	 @classmethod
    def get_data(cls, request, q, columns):
        records = []
        for record in q:
            data_cols = {col: getattr(record, col) for col in columns}
            records.append(data_cols)
        return records

	 @classmethod
    def get(cls,id,request):
		sch = GetDataModelOut()
		q = request.state.session.query(cls).filter(cls.id == id)
		data = cls.get_data(request, q, ['id','name','sal'])
		for row in data:
            sch.message = "Record found successfully"
            sch.result = [row]
            return sch.__dict__

		
@ROUTER.get("/{emp_id}/")
async def get_employee(emp_id: int, request: Request):
    emp = Emp.get(emp_id, request=request)
    return emp
```

### Description

we are using FAST API to develop the APIs and sql alchemy as ORM. we deployed server in the AWS ubuntu instance host, using gunicorn as web server, nginx redirecting request to gunicorn server. few APIs are behaving strange. For example the API has to return id, name, salary parameters in the response. When I hit that API multiple times,  name and id attributes are missing in the result parameter of response for few calls. looking this is the strange behavior from fast API. Any one facing the similar issue. Thanks for your answers in advance. we are not able to reproduce this issue in local system.

### Operating System

Linux

### Operating System Details

ubuntu on AWS VM

### FastAPI Version

0.64

### Python Version

3.9

**Comment by yinziyan1206** [comment]
I guess the problem is not in fastapi but in sqlalchemy. Can you show more details?

**Reply by Samba-Odoo-Apps** [reply]
are you expecting the session configuration details?
SessionLocal = scoped_session(
    sessionmaker(autocommit=False, autoflush=False, bind=engine,
                 expire_on_commit=False))

@contextmanager
def session_scope():
    """Provide a transactional scope around a series of operations."""
    session = SessionLocal()
    try:
        yield session
        session.commit()
    except Exception as err:
        session.rollback()
        raise err
    finally:
        session.close()

**Reply by Samba-Odoo-Apps** [reply]
q = request.state.session.query(cls).filter(cls.id == id)
data = cls.get_data(request, q, ['id','name','sal'])

**Reply by yinziyan1206** [reply]
You can try parsing the results to pydantic model instead of `dict`. They will be validated by fastapi if the attributes are still missing
